superset-frontend/src/setup/setupApp.ts

```diff
@@ -67,7 +67,7 @@ export default function setupApp() {
     ) {
       ev.preventDefault();
       SupersetClient.get({
-        endpoint: ev.currentTarget.href,
+        url: ev.currentTarget.href,
```

Comment:
This is also in #9215.

Reply:
let's revert this change from this pr then if it's in a different one already